packages/server/src/tests/banners/guardianWeeklyPropensityData.ts

```diff
@@ -0,0 +1,24 @@
+import { logInfo } from '../../utils/logging';
+import { streamS3DataByLine } from '../../utils/S3';
+import { isProd } from '../../lib/env';
+
+const guardianWeeklyHighPropensityIds: Set<string> = new Set<string>();
```

Comment:
this file just streams a list of browserIds from S3 into an in-memory Set, just as we did with the single contributor model AB test a while back